### PR TITLE
User more compact Thymeleaf constructs

### PR DESCRIPTION
**Pull request by Shredder121**
Also includes a few accessibility improvements related to those constructs.
- Input nested in label doesn't need `for`.
- Typoes or wrong `for` attribute value

### CODE REVIEW

File: src/main/resources/templates/cla/icla/sign.html [hunk 1]
```diff
@@ -29,16 +29,16 @@
 				</div>
 			</div>
 			<form id="sign-form" class="hidden-print" th:if="${!signClaForm.signed}" th:action="@{/sign/{claName}/icla#sign-form(claName=${signClaForm.claName})}" method="post" th:object="${signClaForm}">
-				<div class="form-group" th:class="${#fields.hasErrors('name')}? 'has-error form-group' : form-group">
+				<div class="form-group" class="form-group" th:classAppend="${#fields.hasErrors('name')}? 'has-error'">
```
Comment (Shredder121) [inline src/main/resources/templates/cla/icla/sign.html]: One moment, spotted something I did wrong.

Comment (Shredder121) [inline src/main/resources/templates/cla/icla/sign.html]: Fixed.

**Comment by mp911de** [comment]
Thanks for the PR. Squashed, merged and polished with eb63974fdf24225e652f6cdd4647dec1d7dd5457.

**Comment by Shredder121** [comment]
Yeah, nice!
I forgot that `#ids` also existed.
Now it's pure awesome!

**Pull request closed without merging**